src/wallet/wallet2.cpp

```diff
@@ -3604,74 +3794,54 @@ std::string wallet2::make_multisig(const epee::wipeable_string &password,
     else
     {
       THROW_WALLET_EXCEPTION_IF(public_keys[i] == local_pkey, error::wallet_internal_error,
-          "Found local spend public key, but not local view secret key - something very weird");
+        "Found local spend public key, but not local view secret key - something very weird");
```

Comment:
Is your editor munging these automatically ? Must be a setting to stop it.

Reply:
Yes, it's editor's fault. The project uses different identations and my IDE goes crazy about it